Spock-core/src/Web/Spock/Internal/Config.hs

```diff
@@ -0,0 +1,23 @@
+{-# LANGUAGE RankNTypes #-}
+module Web.Spock.Internal.Config where
+
+import Data.Word
+import Network.HTTP.Types.Status
+import Web.Spock.Internal.CoreAction
+import qualified Web.Spock.Internal.Wire as W
+
+
+data SpockConfig
+    = SpockConfig
+    { sc_maxRequestSize :: Maybe Word64
+      -- ^ Maximum request size in bytes
+    , sc_errorHandler :: Status -> W.ActionCtxT () IO ()
+      -- ^ Error handler
```

Comment:
Maybe add a comment that the correct status code header is already set (but can be overwritten using `setStatus`)